src/deroff.rs

```diff
@@ -761,7 +761,14 @@ impl Deroffer {
     //             return self.esc()
 
     fn number<'a>(&mut self, s: &'a str) -> Option<&'a str> {
```

Comment:
Why not just return a `bool` like https://github.com/fish-shell/fish-shell/blob/master/share/tools/deroff.py#L603-L610? And I believe it should not take in `s` as argument.

It would be enough to just skip the numbers like the rest. The `unimplemented!()` could be removed in this case. It would also be better to use `mat` instead of `possible` like how the `regex` crate does (and most of how it was implemented in other parts).

Reply:
Any news on this?

Reply:
Sorry I have taken so long to get back to this.

Regarding the function signature, I'd have to defer to others; I didn't make the choice. There seems to be a set of functions of this style, and another set of functions which have signatures which return bool, in the same way the original Python methods do.

For the other parts of this comment, could I ask you to break up into more parts to help make it clearer what you're talking about?

Reply:
Try to use `bool` since we are starting to make patches to change the `Option<&'a str>` to `bool`.

Reply:
I guess we can just merge this first and fix it later.